packages/core/src/Stores/notification-store.js

```diff
@@ -349,10 +349,10 @@ export default class NotificationStore extends BaseStore {
         if (is_logged_in) {
             if (isEmptyObject(account_status)) return;
             const {
-                authentication: { document, identity, income, needs_verification, ownership },
+                authentication: { document, identity, income, needs_verification, ownership } = {},
                 status,
                 cashier_validation,
-            } = account_status;
+            } = account_status || {};
```

Comment:
On the line above you do not allow account_status to be empty object. Here you explicitly treat it like that.

Reply:
Ah yes, overlooked the above condition. Corrected 💯